pytest_django/compat.py

```diff
@@ -1,4 +1,5 @@
 # This file cannot be imported from until Django sets up
+from django.db.backends.base.base import BaseDatabaseWrapper
```

Comment:
Is this (`pytest_django.compat`) a public API?  If not, we could remove it and adjust the import in `plugin.py`.